web/static/index.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+
+    <title>Optimizely Agent</title>
+    <meta name="description" content="Optimizely Agent">
+    <meta name="author" content="Optimizely">
+
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+
+<body>
+<h1>Welcome to Optimizely Agent!</h1>
+<pre>
+                            ,s##########Ms,
+                        ,####################M,
+                      s#########################s
+                    ;#############################m
+                   ################MM##############M
+                  @#@@########"         "###########M
+              ,s#@@@#b#@@@@@M             %##########p
+           ;#####@#|#@@@@@@#               @#########S
+         #######"|#@@@@@@@#b               |##########
+       {######`  ]@@@@@@##|#               ###########
+      ######`  ,#5@@@@#U|#@@@Q           ,###########
+     @###M   ;###^7@#|l#@@@@@@@m,    ,,#############`
+    |##"   ;###"   '#l@@@@@@@#ll#@@@@@@@###########`
+    5"   s#####      @@@@@@#|#l@@@@@@@#bl########M
+      ,#######"        %##S#|@@@@@@@#|#@@######"
+    .#######^            '"#@@@@@##|#@@@####"`
+     #####   ,##m           ;|5#T"*@###5\
+     '#M   ;######M     ,;####`  ,#####
+         ;######"   ;######M   ;######`
+        \#####"   s######"   ;######"
+          "#^  ,#######"   s######"
+              %######`  ,#####M"
+                  ``   """`
+</pre>
+Please refer to the <a href="/openapi.yaml">openapi.yaml</a> spec for API details.
```

Comment:
Cool, I think this is good to keep as one link. We can evaluate whether we need more links, but I actually think keeping this lean is important for now so we don't duplicate documentation efforts across different surfaces